src/App.js

```diff
@@ -1,25 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```

Comment:
- [ ] As it is a requirement for this project, your `App.js` component is suppose to be `class-based`, please kindly fix that.